.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+/test/
```

Comment:
This folder should not be on origin. The purpose of this folder is to test code on local repository only.